documentation/en/installation.md

```diff
@@ -163,7 +163,7 @@ git checkout <release version>
    3.  By default, a ‘multicore-sdr’ option is used in the proofs library. This feature is also used in FFI unless explicitly disabled. To disable building with the ‘multicore-sdr’ dependency, set `FFI_USE_MULTICORE_SDR` to `0`:\
 
 
-       ```shell
+    ```shell
```

Comment:
This will misalign the docs.

<img width="829" alt="Screenshot 2024-08-15 at 12 48 41 PM" src="https://github.com/user-attachments/assets/4b4d8880-8712-45db-8d99-20083d498bd5">